Source/Core/Core/WiiRoot.cpp

```diff
@@ -85,11 +85,15 @@ static bool CopyNandFile(FS::FileSystem* source_fs, const std::string& source_fi
 
   auto source_handle =
       source_fs->OpenFile(IOS::PID_KERNEL, IOS::PID_KERNEL, source_file, IOS::HLE::FS::Mode::Read);
+  // Do not create an empty file on the destination filesystem if the source file doesn't exist
+  if (!source_handle)
+    return false;
```

Comment:
Returning false results in a warning being logged, even though this is a normal scenario. Do we want that?

Reply:
Also, the `CreateFullPath` call should probably be moved to just before the `CreateAndOpenFile` call.

Reply:
Good points, I've made it return true and moved the CreateFullPath call